Test/Test.Neon.Kube/KubeHelper/Test_Json.cs

```diff
@@ -49,7 +49,7 @@ public void SerializeEnum()
 
             serializerOptions.Converters.Add(new JsonStringEnumMemberConverter());
 
-            var t = DataRaidGroupType.stripe;
+            var t = HostingCapabilities.Pausable;
```

Comment:
Is this right?

Reply:
I changed the SerializeEnum() to test against an actual enum like the method name suggests. Now that DataRaidGroupType is no longer an enum, and changed SerializeEnum() to test a random enum I found in the source code. I wasn't sure how to run the test so I'm not sure if it works.